113.md

```diff
@@ -0,0 +1,141 @@
+NIP-113
+===
+Activity Events
+----------------
+`draft` `optional`
+
+This NIP defines `kind:30100`: a parameterized replaceable event to describe activities. Activities are things that one might record via a tracking watch (garmin, fitbit, apple watch, etc...) and wish to share with other.
+
+In general, activities are notes that have an activity file associated with them (gpx, tcx, fit), and a summary can be produced about this activity.
+
+The structure of an activity follows very closely with [NIP-23](23.md) long-form content events.
+### Content
+The `.content` field MAY contain a description of the activity
+### Activity Types
+Activities MUST include a `t` tag with an activity type (defined in Appendix A). e.g. `run | walk | hike`
+### Activity File
+Activities MAY include an `r` tag, which will include a URL to the location where an activity file can be found.
+### Metadata
+`t` - The activity type  
+`r` - location of activity file (OPTIONAL)  
+`title` - The title of the activity  
+`recorded_at` - The unix timestamp (in seconds) when the activity was recorded  
+`encrypted` - An indication that this event is encrypted (OPTIONAL)
+
+```json
+{
+  "kind": 30100,
+  "content": "<description-of-activity>",
+  "tags": [
+	 ["title", "Title of Activity"],
+	 ["t", "<activity-type-from-nip>"]
+	 ["d", "UUID generated by client"],
+	 ["r", "url to download activity file"],
+	 ["published_at", "<unix-timestamp>"],
+	 ["updated_at", "<unix-timestamp>"],
+	 ["recorded_at", "<unix-timestamp>"],
```

Comment:
Is this the timestamp of when the activity has started to record? Or saved / ended? As the whole recording is in most cases a timedelta.